app/src/main/java/de/xikolo/controllers/course/ItemListAdapter.kt

```diff
@@ -46,6 +46,12 @@ class ItemListAdapter(private val section: Section, private val listener: OnItem
 
         holder.textIcon.setText(item.iconRes)
 
+        if (item.timeEffort > 0) {
+            holder.duration.text = item.formatTimeEffort()
+        } else {
+            holder.duration.visibility = View.GONE
+        }
```

Comment:
RecyclerView elements might be reused by the Adapter. This can lead to Views being `GONE` initially.
Please add `holder.duration.visibility = View.VISIBLE` in the first branch.

Reply:
Good to know, thanks! Is fixed